homeassistant/components/plex/media_browser.py

```diff
@@ -132,12 +132,13 @@ def playlists_payload():
             "children": [],
         }
         for playlist in plex_server.playlists():
-            if playlist.playlistType != "audio" and platform == "sonos":
-                continue
-            try:
-                playlists_info["children"].append(item_payload(playlist))
-            except UnknownMediaType:
-                continue
+            if playlist.type != "directory":
+                if playlist.playlistType != "audio" and platform == "sonos":
```

Comment:
Remove the `if playlist.type != "directory":`, add 

```python
if playlist.type != "directory" and playlist.playlistType != "audio" and platform == "sonos":
```

instead. This makes sense in my head at least. That would make the code cleaner imo.

Reply:
Thanks @mawoka-myblock for the suggestion. Consolidated the conditionals to one line. Tested and works.